data/json/items/resources/misc.json

```diff
@@ -33,4 +33,20 @@
     "flags": [ "NONCONDUCTIVE" ],
     "use_action": { "type": "deploy_furn", "furn_type": "f_floor_mattress" }
   }
+  {
+    "id": "down_mattress",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "down_mattress",
+    "name_plural": "down_mattresses",
+    "description": "This is a single, or twin, sized down filled mattress.",
+    "weight": 20000,
+    "volume": "300000 ml",
+    "price": 1000,
+    "material": [ "cotton" ],
+    "symbol": "0",
+    "color": "white",
+    "flags": [ "NONCONDUCTIVE" ],
```

Comment:
Not needed; cotton material is already nonconductive.

Reply:
Changed in github should be resolved